cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-renewed.cy.js

```diff
@@ -0,0 +1,316 @@
+          loansData.forEach(({ id, item }) => {
+            if (item.title.includes('InstanceForDeclareLost')) {
+              UserLoans.declareLoanLostViaApi({
+                servicePointId: testData.userServicePoint.id,
+                declaredLostDateTime: moment.utc().format(),
+              }, id);
+            }
+          });
+        });
+
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    cy.getToken(userData.username, userData.password);
+    UserLoans.updateTimerForAgedToLost('reset');
+    cy.getAdminToken();
+    cy.get('@items').each((item) => {
+      CheckInActions.checkinItemViaApi({
+        itemBarcode: item.barcode,
+        servicePointId: testData.userServicePoint.id,
+        checkInDate: new Date().toISOString(),
+      });
+    });
+    NewFeeFine.getUserFeesFines(userData.userId).then((userFeesFines) => {
+      const feesFinesData = userFeesFines.accounts;
+      feesFinesData.forEach(({ id }) => {
+        cy.deleteFeesFinesApi(id);
+      });
+    });
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    PaymentMethods.deleteViaApi(testData.paymentMethodId);
+    UsersOwners.deleteViaApi(testData.ownerId);
+    cy.deleteLoanPolicy(loanPolicyBody.id);
+    LostItemFeePolicy.deleteViaApi(lostItemFeePolicyBody.id);
+    CirculationRules.deleteRuleViaApi(originalCirculationRules);
+    cy.deleteLoanType(testData.loanTypeId);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    Conditions.resetConditionViaApi('cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a', 'Maximum outstanding fee/fine balance');
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+  });
+  it(
+    'C350655 Verify automated patron block "Maximum outstanding fee/fine balance" removed after lost item renewed (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      cy.visit(SettingsMenu.conditionsPath);
+      Conditions.waitLoading();
+      Conditions.select('Maximum outstanding fee/fine balance');
+      Conditions.setConditionState(blockMessage);
+      cy.visit(SettingsMenu.limitsPath);
+      Limits.selectGroup(patronGroup.name);
+      Limits.setLimit('Maximum outstanding fee/fine balance', '624');
+      // needed for the "Lost Item Fee Policy" so items can get "aged to lost" status
+      cy.wait(230000);
```

Comment:
It seems too long
Is it possible to decrease this value?
Do you really need about 4 min pause of test run?

Reply:
Unfortunately, this is the bare minimum. without going into details, it works according to the following principle:
Check out = 1:52 PM  >>  Overdue = 1:53 PM  >> Aged to lost + billing configured for 1:54 PM  >>  Aged to lost appears at 1:35 PM  >>  Fees/fine appear at approx. 1:56 PM.